CONTRIBUTING.md

```diff
@@ -196,6 +196,15 @@ This allows you to make changes to your local vireo and have them be instantly a
 
 **WARNING** Since the local VireoSDK is symlinked to the JavaScript application be aware of tooling that could accidently delete or modify your local development files unexpectedly.
 
+If you have a .NET application that depends on Vireo you can do the following (you must have nuget.exe available)
+- In the directory with the `VireoSDK.nuspec` file run:
+  ```console
+  nuget pack VireoSDK.nuspec -properties version="[version]"
+  ```
+  
+  replacing \[version\] with some version.
+- See [installing a nuget package](https://docs.microsoft.com/en-us/nuget/consume-packages/ways-to-install-a-package) for the various ways to install the created package to your .NET application.
+
 ### Testing on a different machine / CI
 If you want to test your local vireo changes on a separate machine from your local machine (a CI or test machine) without publishing to the npm registry you can do the following:
```

Comment:
So if you create a vireo.nupkg file locally, how do you create a build that points to it on the CI? Please add instructions to this section as well